test_app/tests/lib/utils/test_models.py

```diff
@@ -309,3 +309,43 @@ def test_diff_sanitizes_encrypted_fields_removed(disable_activity_stream):
     assert delta.removed_fields['testing1'] == ENCRYPTED_STRING
     assert delta.removed_fields['testing2'] == ENCRYPTED_STRING
     assert 'message' not in delta.removed_fields
+
+
+@pytest.mark.parametrize(
+    "username,expected_value",
+    [
+        (None, False),
+        ("system", True),
+        ("random", False),
+    ],
+)
+def test_is_system_user_system_user_setting_set(username, expected_value, system_user, random_user):
+    if username is None:
+        user = None
+    elif username == 'system':
+        user = system_user
+    else:
+        user = random_user
```

Comment:
I find myself doing this kind of thing often in parametrized tests where I end up usaing a magic string to pick between fixtures, and I hate it. I wish there were a cooler syntax for this kind of thing.

Reply:
Yea, it would be nice if you could put a fixture in the parametrized data itself or somehow have a builtin way to refer to a fixture.